src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.DivRem.cs

```diff
@@ -321,7 +321,7 @@ private static uint[] CreateCopy(uint[] value)
             Debug.Assert(value != null);
             Debug.Assert(value.Length != 0);
 
-            var bits = new uint[value.Length];
```

Comment:
BTW, we are totally fine with var when it's clear what the resulting type is. We just don't like var x = SomeFunction();